README.md

```diff
@@ -1,26 +1,49 @@
 # JS Pokemon Search Assignment
 
+![pikachu](https://media.giphy.com/media/uLnPIWsqIz2aA/giphy.gif)
+
 ## Objectives
 
 - DOM Manipulation
 - Events and Event Handlers
 - Callbacks
-- Constructors and Prototypes or ES6 Classes (For part two)
+- Constructors and Prototypes or ES6 Classes (optional)
+
+---
+
+Hello, your assignment today is to re-create the functionality of our
+[Pokemon search engine](https://pokemon-search.netlify.com/).
 
-Hello, your assignment today is to re-create the functionality of our [Pokemon search engine](https://pokemon-search.netlify.com/).
+p.s. Don't forget to include the ability to toggle the card image and reset the card image upon submission of a new
+search.
 
-p.s. Don't forget to include the ability to toggle the card image and reset the card image upon submission of a new search.
+---
 
 ## Instructions
 
-We're building out a search feature in our application (no backend persistence). A user should be able to search for a Pokemon and flip that Pokemon card to see the back side.
+- We're building out a search feature in our application (no backend persistence).
+
+- A user should be able to search for a Pokemon and flip that Pokemon card to see its alternate sprite.
+
+- Two files containing the same information are included: `db.json` and `pokemon.js`. If you've learned `fetch`,
+  consider using `json-server` to spin up a simple RESTful API that will give you your pokemon data:
+  - `$ npm install -g json-server`
+  - `$ json-server --watch db.json`
+
```

Comment:
We could consider adding how to import / require the data in pokemon.js if they have not yet used fetch.

Reply:
perf. Updated the README for students that do not know `fetch` yet